app/docs/0.12.x/admin-api.md

```diff
@@ -1787,6 +2025,111 @@ Attributes | Description
 HTTP 204 No Content
 ```
 
+---
+
+### Set target as healthy
+
+Set the current health status of a target in the load balancer to "healthy".
+This resets the health counters of the health checkers running in all workers
+of the kong node, and broadcasts a cluster-wide message so that this
+health status is propagated to the whole Kong cluster.
```

Comment:
perhaps a link to docs on the kong cluster invalidations mechanism, since this is not an instantaneous change?

Reply:
Adding the link!